staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -155,6 +156,7 @@ func NewCmdCp(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.C
 	}
 	cmdutil.AddContainerVarFlags(cmd, &o.Container, o.Container)
 	cmd.Flags().BoolVarP(&o.NoPreserve, "no-preserve", "", false, "The copied file/directory's ownership and permissions will not be preserved in the container")
+	cmd.Flags().IntVarP(&o.MaxTries, "retries", "t", 20, "Set number of retries to complete a copy operation from a container. Specify 0 to disable or any negative value for infinite retrying. The default is 20.")
```

Comment:
I'd prefer this to be opt-in for starters, so default to be 0, and don't use `-t` as the shorthand, since that's being used elsewhere as shorthand for `--tty` so that might be confusing here a bit.